docs/src/getting-started/building/source.md

```diff
@@ -43,9 +43,28 @@ $ sudo apt install -y libhwloc-dev \
     curl
 ```
 
+### CUDA
+
+Running `polka-storage-provider-client` & `polka-storage-provider-server` via CUDA gives significant performance boosts
+as parts of the proving algorithm can be sent-off to the GPU. By default components are compiled with OpenCL.
```

Comment:
Q: is there a way to ensure that OpenCL uses the GPU? According to what Luca said in Slack, it runs on the CPU by default (?)

Reply:
If there is OpenCL Graphics driver on your machine, it'll use GPU. That's what it did in my case. 
It runs on CPU if there is no GPU, at least in the implementation we're using. That's why it works by default.